From: Joe Nahmias <joe@nahmias.net>
Date: Sat, 12 Sep 2020 22:20:56 -0400
Subject: use local intershpinx inventory files

Forwarded: not-needed
---
 docs/source/conf.py | 25 ++++++++++++++++++++-----
 1 file changed, 20 insertions(+), 5 deletions(-)

diff --git a/docs/source/conf.py b/docs/source/conf.py
index a5b6818..dc257a8 100644
--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -359,11 +359,26 @@ texinfo_documents = [
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
-    'python': ('https://docs.python.org/3/', None),
-    'ipython': ('https://ipython.readthedocs.io/en/stable/', None),
-    'pymongo': ('https://pymongo.readthedocs.io/en/stable/', None),
-    'distributed': ('https://distributed.readthedocs.io/en/stable/', None),
-    'jupyterclient': ('https://jupyter-client.readthedocs.io/en/stable/', None),
+  'python': (
+    'https://docs.python.org/3/',
+    ('/usr/share/doc/python3-doc/html/objects.inv', None)
+  ),
+  'ipython': (
+    'https://ipython.readthedocs.io/en/stable/',
+    ('/usr/share/doc/python-ipython-doc/html/objects.inv', None)
+  ),
+  'pymongo': (
+    'https://pymongo.readthedocs.io/en/stable/',
+    ('/usr/share/doc/python-pymongo-doc/html/objects.inv', None),
+  ),
+  'distributed': (
+    'https://distributed.readthedocs.io/en/stable/',
+    ('/usr/share/doc/python-distributed-doc/html/objects.inv', None)
+  ),
+  'jupyterclient': (
+    'https://jupyter-client.readthedocs.io/en/stable/',
+    ('/usr/share/doc/python-jupyter-client-doc/html/objects.inv', None)
+  ),
 }
 
 import os
